src/apiCalls.tsx

```diff
@@ -1,23 +1,19 @@
+import { RandomCocktail } from './Definitions/RandomCocktail'
 const rootUrl = "https://www.thecocktaildb.com/api/json/v1/1";
 
 //random cocktail: https://www.thecocktaildb.com/api/json/v1/1/random.php
 
 //lookup by id: https://www.thecocktaildb.com/api/json/v1/1/lookup.php?i=11007
 
 
-export const getRandomCocktail = async () => {
+export const getRandomCocktail = async ():Promise<RandomCocktail> => {
 	const response = await fetch(`${rootUrl}/random.php`);
-	const data = await response.json();
-  const filteredData = data.drinks.filter((drink: string) => drink !== null);
+  const data = await response.json();
 
-  return filteredData[0];
-  // if (response.ok) {
-  //   return await response.json();
-  // } else {
-  //   throw new Error({...response})
-  // }
+  return data.drinks[0]
 }
 
+//Add AlcoholicCocktail[] to definitions 
 export const getAllCocktails = async () => {
```

Comment:
sorry I completely forgot to add a definition to this async function

Reply:
No worries - We can all get practice writing & adding definition to the various calls we have to make.